app/helpers/sharing_permissions_helper.rb

```diff
@@ -1,225 +1,51 @@
 module SharingPermissionsHelper
 
-  ITEMS_NOT_IN_ISA_HASH = {
-      "id": "not_isa-tree",
-      "data": {
-          "loadable": false
-      },
-      "li_attr": {
-          class:"root-node"
-      },
-      "a_attr": {
+  def get_permission_list (item)
```

Comment:
Is this method used in more than one place?

I feel like it is doing too much HTML rendering, and might be better in a partial (which would make it more readable), maybe with some small helper methods e.g. to generate the policy text for each permission.

Reply:
Addressed in [8c79844](https://github.com/seek4science/seek/pull/1551/commits/8c79844630b63c4bda68fb657d2c855f62919260)